test/esm/package.json

```diff
@@ -2,7 +2,7 @@
   "private": true,
   "name": "@babel-internal/test-esm",
   "devDependencies": {
-    "@babel/core": "^7.13.10",
+    "@babel/core": "^7.17.8",
```

Comment:
I guess this somehow didn't get updated in #122 ? I believe there's a similar line in the `yarn.lock` from this